src/theme.tsx

```diff
@@ -1719,6 +1719,24 @@ export const theme = createTheme({
 				}),
 			},
 		},
+		MuiMenuItem: {
+			styleOverrides: {
+				root: {
+					'&.Mui-selected': {
+						background: color.bg.active.value,
+						'&:hover': {
+							background: color.bg.active.value,
+						},
+					},
+					'&:hover': {
+						background: color.bg.hover.value,
+					},
+				},
+				selected: {
+					background: color.bg.active.value,
+				},
+			},
+		},
```

Comment:
This will affect Autocomplete, Select, and DropdownButton (and of course any other uses of MenuItem). Just want to make sure it's intended to affect *all* of these

Reply:
it's intended :)